docs/tutorials/add-route.md

```diff
@@ -106,7 +106,7 @@ App::post('/v1/storage/buckets/:bucketId/files')
 ```
 
 #### Events
-* event - A pattern that is associated with the route in behalf of realtime messaging.
+* event - A pattern that is associated with the route in behalf of real time messaging.
```

Comment:
We consider `realtime` the correct spelling in our code base :)

Reply:
reverted back